packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-dialog.tsx

```diff
@@ -6,8 +6,11 @@ import ContractTypeMenu from './ContractTypeMenu';
 
 type TContractTypeDialog = {
     is_info_dialog_open: boolean;
-    onClose: React.ComponentProps<typeof MobileDialog>['onClose'];
+    onDescriptionClose: () => void;
```

Comment:
renaming feels unnecessary.

Reply:
Will rename back. It will be closed, when user click on cross icon in description (on desktop + mobile)